pkgs/development/compilers/lingua-franca/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, pkgs, stdenv, fetchFromGitHub, jdk11_headless, makeWrapper }:
+let
+  version = "0.1.0";
+in
+stdenv.mkDerivation {
+  name = "lfc";
+  version = version;
+
+  src = fetchTarball {
```

Comment:
Please use fetchzip or fetchurl.

Reply:
Just out of curiosity what is the problem with fetchTarball ?